packages/transaction-controller/src/helpers/PendingTransactionTracker.test.ts

```diff
@@ -263,6 +273,26 @@ describe('PendingTransactionTracker', () => {
       expect(failTransaction).toHaveBeenCalledTimes(0);
       expect(transactionsListener).toHaveBeenCalledTimes(0);
     });
+
+    it('logs error while checking the status of a submitted transaction', async () => {
```

Comment:
A pattern for us to decide on, but typically I don't create unit tests for logging as it's not true business logic, but rather just to aid debugging so could bloat the test file if we did it for every instance.

That said, we could adapt this unit test to instead just ensure we don't throw in the event listener if the query fails?